sklearn/linear_model/_cd_fast.pyx

```diff
@@ -622,8 +622,8 @@ def enet_coordinate_descent_gram(floating[::1] w,
 
 def enet_coordinate_descent_multi_task(
         floating[::1, :] W, floating l1_reg, floating l2_reg,
-        np.ndarray[floating, ndim=2, mode='fortran'] X,  # TODO: use views in 0.24
-        np.ndarray[floating, ndim=2, mode='fortran'] Y,
+        np.ndarray[floating, ndim=2, mode='fortran'] X,  # TODO: use views with Cython 0.3
```

Comment:
I think its cython 3.0 when we can do this.

```suggestion
        np.ndarray[floating, ndim=2, mode='fortran'] X,  # TODO: use views with Cython 3.0
```

Reply:
right they went from 0.29 to 3.0 :)

Reply:
This is even worst than moving from 0.24 to 1.0....

Reply:
It's fine because `(3, 0) > (0, 29)` :)

Going from 0.29 to 0.3 would have be problematic on the other hand because `(0, 3) < (0, 29)`.